### PR TITLE
[master] Update is_enabled function to show global status

### LINKED ISSUE #51823: shedule.is_enabled should accept no argument
Allowing to check the current state of `schedule.enable`/`disable`.

### PR DESCRIPTION
**Pull request by garethgreenaway**
### What does this PR do?
Update is_enabled to make name argument optional, when empty is_enabled will indicate if the scheduler is enabled or disabled.  Add test for is_enabled.

### What issues does this PR fix or reference?
#51823

### Tests written?
**[NOTICE] Bug fixes or features added to Salt require tests.**
Please review the [test documentation](https://docs.saltstack.com/en/latest/topics/tutorials/writing_tests.html) for details on how to implement tests into Salt's test suite.
Yes

### Commits signed with GPG?
Yes

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: doc/ref/configuration/minion.rst [hunk 1]
```diff
@@ -831,12 +831,28 @@ Default: ``False``
...
-to enable set grains_cache to ``True``.
+to enable set ``grains_cache`` to ``True``.
 
 .. code-block:: yaml
 
     grains_cache: False
 
+.. conf_minion:: grains_cache_expiration
+
+``grains_cache_expiration``
```
Comment (Ch3LL) [inline doc/ref/configuration/minion.rst:842]: this seems to not be related. Was this added on purpose?

Comment (garethgreenaway) [reply doc/ref/configuration/minion.rst:842]: Nope.  Not sure how that got there.

Comment (garethgreenaway) [reply doc/ref/configuration/minion.rst]: Updating to latest master resolved this.

**Review by Ch3LL — DISMISSED** [review]

File: salt/modules/schedule.py [hunk 2]
```diff
@@ -157,24 +157,32 @@ def list_(show_all=False, show_disabled=True, where=None, return_yaml=True):
         return {"schedule": {}}
 
 
-def is_enabled(name):
+def is_enabled(name=None):
     """
     List a Job only if its enabled
 
+    If not job is specified, indicate
```
Comment (dwoz) [inline salt/modules/schedule.py:164]: It's not clear to me what this doc string means

**Review by dwoz — CHANGES_REQUESTED** [review]

**Review by dwoz — APPROVED** [review]

**Pull request merged**